modules/ROOT/pages/platform/logging/query-log-analyzer.adoc

```diff
@@ -0,0 +1,74 @@
+[[aura-monitoring]]
+= Query log analyzer
+
+label:AuraDB-Professional[]
+label:AuraDB-Enterprise[]
+
+Query log analyzer is a feature that provides a UI to review the queries executed on an Aura instance.
+
+To access *Query log analyzer*:
+
+. Navigate to the https://console.neo4j.io/?product=aura-db[Neo4j Aura Console] in your browser.
+. Select the instance you want to access.
+. Select the *Logs* tab.
+. Select the *Query log analyzer* button.
+
+Query log analyzer is split up in two parts, a timeline and a table with two tabs:
+
+* *Query timeline* - Timeline showing metrics for number of queries, failed queries and query latency.
+* *Summary* - An aggregated view of query logs, giving a high level overview over the selected time period.
+* *Details* - A detailed view showing individual query executions in the selected time period. 
+
+To fetch logs, first choose a time range in the Query timeline. 
+With a time selection done, press the *Fetch logs* button. 
+You may optionally choose any filters or search text if required, then press *Go*.
+
+A summary of query executions is returned, showing aggregations per query.
+To see the individual query executions, click the right arrow at the end of the line to show details for that query.
+The details pane shows individual executions.
+
+== Chart interactions
```

Comment:
Should we mention that the entire chart can be collapsed to give more screen estate to the table? 🤔